### PR TITLE
Add didimputation

### PR DESCRIPTION
**Pull request by okankcb**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/r-didimputation/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    - r-data.table >=1.10.0
+    - r-dplyr
+    - r-fixest >=0.10.0
+    - r-glue
+    - r-magrittr
+    - r-purrr
+    - r-stringr
+    - r-tidyr
+  run:
+    - r-base
```
Comment (FernandezMathieu) [inline recipes/r-didimputation/meta.yaml:49]: Ajoute la ligne
`- libopenblas`
en dessous de la ligne r-base

File: recipes/r-didimputation/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/didimputation_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/didimputation/didimputation_{{ version }}.tar.gz
+  sha256: 0307b820d9103f367130f436f12143db140c226efb3ca18d59b2b829c7215f88
+
+build:
+  merge_build_host: True  # [win]
```
Comment (FernandezMathieu) [inline recipes/r-didimputation/meta.yaml:16]: Ajoute 
`skip: true  # [win]`
en dessous de build: avec la même indentation que `merge_build_host`

Comment (FernandezMathieu) [reply recipes/r-didimputation/meta.yaml:16]: Finalement essaye 
```
skip: true # [win or osx]
```

**Comment by okankcb** [comment]
@conda-forge-admin

**Comment by okankcb** [comment]
@conda-forge/help-python The recipe has passed all checks and is ready for review. Thanks.

**Comment by FernandezMathieu** [comment]
@conda-forge/core Please can you merge this ? thx.

**Comment by FernandezMathieu** [comment]
@dopplershift hello sorry to ping you directly but you have validated our last PR : https://github.com/conda-forge/staged-recipes/pull/21783#event-8324276694 

Can you please merge this PR ?

**Comment by dopplershift** [comment]
Sorry, I don't do R. Ping @conda-forge/help-r

**Comment by okankcb** [comment]
@conda-forge/help-r

**Comment by okankcb** [comment]
@conda-forge/help-r

**Comment by okankcb** [comment]
conda-forge/staged-recipes Ready for review

**Comment by okankcb** [comment]
@conda-forge/staged-recipes Ready for review

**Comment by FernandezMathieu** [comment]
Hi @dbast can you merge please ?

**Comment by FernandezMathieu** [comment]
@conda-forge/help-r @conda-forge/staged-recipes Hello seem like this PR have slipped under your radar. It's ready for review! Thanks!

**Comment by jmboehm** [comment]
Hello, would be wonderful if this could be merged-- many thanks!

**Pull request closed without merging**